cmd/goal/README.md

```diff
@@ -3,27 +3,34 @@
 Unless otherwise noted, it is assumed that the working directory
 begins at the top level of the `go-algorand` repo.
 
-It is also assumed that the main README's installation instructions have been followed and `make install` run so that the `goal` executable is available via `${GOPATH}/bin/goal`. 
-You can verify this to be the case by comparing the output of `which goal` with the output of `echo ${GOPATH}/bin/goal`.
+It is also assumed that the main README's installation instructions have been followed and `make install` run so that the `goal` executable has been rebuilt from the same source as this example and is available in the shell environment.
+You can run `ls -l $(which goal)` after your `make install` and look at the installation time as a sanity check.
 
-## Starting a Single Node Dev Network 
+Finally, all the `goal` commands assume that `${ALGORAND_DATA}` has been set. See the first Q/A for how this is done.
+
+## Starting a Single Node Dev Network
 
 ### Q: Having just completed a new build in go-algorand, how do I get a single node dev network up, with algos in an easily accessible wallet from goal?
 
 ### A:
 
 ```sh
-# create a networks directory if you don't already have it: 
-mkdir -p ~/networks
-
-# Set this to where you want to keep the network files (and data dirs will go beneath)
+# set this to where you want to keep the network files (and data dirs will go beneath)
 NETWORKS=~/networks
 
-# set this to "name" your network:
+# create a networks directory if you don't already have it
+mkdir -p ${NETWORKS}
+
+# set this to "name" your network
 NAME=niftynetwork
 
 goal network create -n ${NAME} -r ${NETWORKS}/${NAME} -t ./test/testdata/nettemplates/OneNodeFuture.json
```

Comment:
I suppose you should comment on where this template file is.  This command assumes you're running in the root directory of your go-algorand repo.